osquery/tables/system/windows/windows_crashes.cpp

```diff
@@ -0,0 +1,672 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#define _WIN32_DCOM
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Winternl.h>
+#pragma warning(push)
+// C:\Program Files (x86)\Windows Kits\8.1\Include\um\DbgHelp.h(3190):
+// warning C4091: 'typedef ': ignored on left of '' when no variable is
+// declared
+#pragma warning(disable : 4091)
+#include <DbgHelp.h>
+#pragma warning(pop)
+#include <DbgEng.h>
+
+#include <boost/algorithm/string/predicate.hpp>
+#include <boost/filesystem.hpp>
+
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/wmi.h"
+
+namespace alg = boost::algorithm;
+namespace fs = boost::filesystem;
+
+namespace osquery {
+namespace tables {
+
+const std::string kLocalDumpsRegKey =
+    "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\Windows Error "
+    "Reporting\\LocalDumps";
+const std::string kDumpFolderRegPath = kLocalDumpsRegKey + "\\DumpFolder";
+const std::string kFallbackFolder = "%TMP%";
+const std::string kDumpFileExtension = ".dmp";
+const std::string kSymbolPath =
+    "C:\\ProgramData\\dbg\\sym;"
+    "cache*C:\\ProgramData\\dbg\\sym;"
+    "srv*C:\\ProgramData\\dbg\\sym*https://msdl.microsoft.com/download/symbols";
+const unsigned long kSymbolOptions =
```

Comment:
Make this a local variable in `processDebugEngine`, as currently you've got quite a few global vars and I think this one will be fine local.